### PR TITLE
Gravity and the supermatter count as critical machines

### PR DESCRIPTION
**Pull request by ChangelingRain**
and as such, the grid check event will not abruptly disable gravity and or cause the supermatter to explode

### CODE REVIEW

**Comment by GunHog** [comment]
Awwwwwwwwwwwwwwwwwwww :(

**Comment by Shadowlight213** [comment]
Major :-1: to making gravity critical. If people aren't paying attention, they deserve to lose gravity.

**Comment by Lakart** [comment]
Would rather see the event chance lowered(since it seems to happen quite often) rather than it not making things go to crap. Things SHOULD go to crap when the station suddenly loses all power.

**Comment by Mervill** [comment]
👍 supermatter exploding = unfun bad thing
👎 temporary gravity loss = fun bad thing

**Comment by MimicFaux** [comment]
Mervill has the right of it. Supermatter is a critical piece of machinery. Gravity is an inconvenience that makes the grid check more impactful without actually ruining anything.

File: code/modules/power/apc.dm [hunk 1]
```diff
@@ -666,7 +666,7 @@
 		ui = new(user, src, ui_key, "apc", name, 535, 515, master_ui, state)
 		ui.open()
 	if(ui)
-		ui.set_autoupdate(state = (failure_timer ? 1 : 0))
+		ui.set_autoupdate(state = (failure_timer ? FALSE : TRUE))
```
Comment (Cyberboss) [inline code/modules/power/apc.dm]: Uhhh, did you intend to reverse this behavior?

**Comment by optimumtact** [comment]
Remove the gravity gen check, it's actually quite fun to lose gravity.
